docs/airflow/dags-maintenance.md

```diff
@@ -1,60 +1,59 @@
-| `download_gtfs_schedule_v2`        | Yes   | Don't need to rerun more than once if multiple failures; scraped data is labeled by actual execution time                                                                                             |
-| `parse_and_validate_rt_v2`         | No    |                                                                                                                                                                                                       |
-| `payments_loader`                  | Yes   | This is deprecated, pending data consumers switching to v2 models. See below for maintenance.                                                                                                         |
-| `scrape_feed_aggregators`          | Yes   | Scraped data is labeled by actual execution time                                                                                                                                                      |
-| `transform_warehouse`              | Yes   | Configurable via JSON task parameters to run a portion of the DAG (`dbt_select_statement` key); outputs timestamped artifacts                                                                         |
-| `transform_warehouse_full_refresh` | Yes   | Manual job to execute a full-refresh dbt job; should be used sparingly; configurable via JSON task parameters to run a portion of the DAG (`dbt_select_statement` key); outputs timestamped artifacts |
-| `unzip_and_validate_gtfs_schedule` | No    |                                                                                                                                                                                                       |
-
-### GTFS Realtime parsing and validation
-The primary DAG relevant to GTFS Realtime data is [parse_and_validate_rt_v2](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/parse_and_validate_rt_v2).
-This DAG parses RT files into JSONL and writes them to GCS for querying via BigQuery external tables.
-It also executes the [realtime validator](https://github.com/MobilityData/gtfs-realtime-validator)
-against the RT files and writes the validation notices as JSONL to GCS for querying.
-
-The raw proto files themselves are fetched via [the archiver](https://github.com/cal-itp/data-infra/tree/main/services/gtfs-rt-archiver-v3).
+#### Scheduled vs. ad-hoc
+
+Additionally, DAGs can either be scheduled or ad-hoc:
+
+* **Scheduled** DAGs are designed to be run regularly, based on the [cron schedule](https://airflow.apache.org/docs/apache-airflow/1.10.1/scheduler.html) set in the DAG's `METADATA.yml` file
+* **Ad-hoc** DAGs are designed to be run as one-offs, to automate a workflow that is risky or difficult for an individual user to run locally
```

Comment:
You could add that these are identified by `schedule: None`; also you could indicate that all data interval processing DAGs are Scheduled